app/src/main/java/com/example/musicapp/service/InstrumentService.kt

```diff
@@ -11,31 +11,26 @@ import android.util.Log
 
 class InstrumentService {
 
-    fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
+    internal fun fetchInstruments(): MutableLiveData<ArrayList<Instrument>> {
         var _instruments = MutableLiveData<ArrayList<Instrument>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(MusicAppDAO::class.java)
         val call = service?.getAllInstruments()
-        call?.enqueue(object: Callback<ArrayList<Instrument>>{
+        call?.enqueue(object : Callback<ArrayList<Instrument>> {
 
             override fun onFailure(call: Call<ArrayList<Instrument>>, t: Throwable) {
                 val j = 1 + 1
                 val i = 1 + 1
-
```

Comment:
I just removed the unnecessary blank line here (Not one of my code review suggestions).